docs/pages/chat-apps/core-messaging/disappearing-messages.mdx

```diff
@@ -234,6 +234,12 @@ To learn more about the background worker, see [disappearing_messages.rs](https:
 
 Expired messages don't require manual removal from the UI. If your app UI updates when the local storage changes, expired messages will disappear automatically when the background worker deletes them from local storage.
 
+## Exclude from archive-based backups
+
+You can exclude messages with disappearing message settings from archive-based backups. 
```

Comment:
Maybe it's clearer to say "you can exclude disappearing messages"

Reply:
Done - thank you so much!